.github/workflows/e3sm-gh-ci-cime-tests.yml

```diff
@@ -3,16 +3,6 @@ name: gh
 on:
   pull_request:
     branches: [ master ]
-    paths:
```

Comment:
Why did you remove these? If a PR only touches documentation, it seems pointless to run the CIME tests..

Reply:
Those went in a different PR.

Reply:
Ah, weird. GH showed them to me as part of the changes since my last review. But indeed it doesn't show them when I look for all changes.

Thanks!